lib/util/template/index.ts

```diff
@@ -28,6 +28,14 @@ handlebars.registerHelper('containsString', (str, subStr) =>
 
 handlebars.registerHelper('equals', (arg1, arg2) => arg1 === arg2);
 
+handlebars.registerHelper('includes', (arg1: string[], arg2: string) => {
+  if (arg1?.length && typeof arg2 === 'string') {
```

Comment:
```suggestion
  if (is.array(arg1, is string) && is.string(arg2)) {
```

Reply:
Sure, I can do that :)

Reply:
Raised the MR with the minimal changes for the includes helper https://github.com/renovatebot/renovate/pull/28148 :)